### ISSUE TITLE
The path indicated in `repos-file` (.github/steward-repos.md) does not exist

### ISSUE THREAD

**Issue by slavkof** [scala-steward-org/scala-steward-action#589]
Hello,

Is there an issue when specifying private projects inside repos file? There certainly is not typo, I am specifying our org/repo-name in the file. `- BackendArchitects/BA-Core` 
(repos-file: '.github/steward-repos.md')

Any ideas? Thanks a lot.

**Comment by alejandrohdezma** [comment]
Hey @slavkof! Could you rephrase your question? I'm not sure I understand it

**Comment by slavkof** [comment]
Hi,
Apologies for not being clear. 
Here is my steward-repos.md file
```
- BackendArchitects/BA-Core
- BackendArchitects/wlapihub
- BackendArchitects/ANA-Mobile-API
```
These are all private repositories. The error I get when I run steward action is `the path indicated in repos-file (.github/steward-repos.md) does not exist`. Scala stwarard worked on the first repo (BA-Core) without any issues when I was just running the action without specifying repos-file.

Here is the git action:
```
name: Launch Scala Steward

on:
  workflow_dispatch:
  schedule:
    - cron: '0 0 * * 0'

jobs:
  scala-steward:
    runs-on: ubuntu-22.04
    name: Launch Scala Steward
    steps:
      - name: Launch Scala Steward
        uses: scala-steward-org/scala-steward-action@v2
        with:
          github-app-id: ${{ secrets.APP_ID }}
          github-app-installation-id: ${{ secrets.APP_INSTALLATION_ID }}
          github-app-key: ${{ secrets.APP_PRIVATE_KEY }}
          github-app-auth-only: true
          repos-file: '.github/steward-repos.md'
```

**Comment by alejandrohdezma** [comment]
Oh! I see what's happening here. By default GitHub Actions do not retrieve your repository data. You need to explicitly use the [`actions/checkout`](https://github.com/actions/checkout) action to retrieve your repository in order to access `.github/steward-repos.md`.

**Comment by slavkof** [comment]
AH, thank you so much :)
